components/scream/src/physics/share/physics_constants.hpp

```diff
@@ -32,6 +32,7 @@ struct Constants
   static constexpr Scalar RhoIce        = 917.0;        // Ice density at 0 C from Wallace+Hobbes 1977
   static constexpr Scalar MWH2O         = 18.016;
   static constexpr Scalar MWdry         = 28.966;
+  static constexpr Scalar o2mmr         = 0.23143;      // o2 mass mixing ratio
```

Comment:
I think this should probably an input variable (read from scream_input.yaml) rather than a physics constant. I think this is the O2 mixing ratio that's used to populate the 3d distribution of O2 in the atmosphere. So it's not a fundamental constant.